docs/jax_array_migration.md

```diff
@@ -91,9 +91,9 @@ sharding = jax.sharding.NamedSharding(mesh, P('x'))
 
 sharded_x = jax.device_put(x, sharding)
 
-# `mul_sharded_x` and `sin_sharded_x` are sharded. `jit` is able to operate over a 
+# `dot_sharded_x` and `sin_sharded_x` are sharded. `jit` is able to operate over a
 # sharded array without copying data to a single device.
-mul_sharded_x = sharded_x @ sharded_x.T
+dot_sharded_x = sharded_x @ sharded_x.T
```

Comment:
I wasn't sure what the intention was. Maybe the correction should be to change `@` with `*`.
Dot will mostly show that even if comm is needed to perform the operation, the output stay sharded.
I think it is good to show that more complex case.

Reply:
`@` is matrix multiplication AFAIK.  Maybe matmul_x is more accurate?

Reply:
I amended the commit.